### ISSUE TITLE
Please correct the emulation for where op

### ISSUE THREAD

**Issue by BruceDai** [webmachinelearning/webnn#488]
Current the below emulation is used for **where** op in https://github.com/webmachinelearning/webnn/pull/478,
```
builder.add(
  builder.mul(
    input,
    builder.cast(condition, input.type())),
  builder.mul(
    other,
    builder.cast(builder.not(condition), other.type())));
```
but these **!0** uint8 values of **condition** tensor means **true**, so the first **multiply** operation isn't correct.

@wchao1115 @huningxin @fdwr PTAL, thanks.

**Comment by anssiko** [comment]
Fixed in https://github.com/webmachinelearning/webnn/commit/2d976cb272c77cc4adc81e7405959dec011b8ae3. This issue can be closed when PR #478 lands.

**Comment by BruceDai** [comment]
Close it since #478 has fixed it.
